deny.toml

```diff
@@ -27,7 +27,7 @@ allow-osi-fsf-free = "neither"
 copyleft = "deny"
 # We want really high confidence when inferring licenses from text
 confidence-threshold = 0.93
-allow = ["Apache-2.0", "Apache-2.0 WITH LLVM-exception", "MIT", "MPL-2.0"]
+allow = ["Apache-2.0", "Apache-2.0 WITH LLVM-exception", "MIT", "MPL-2.0", "ISC", "CC0-1.0"]
```

Comment:
> The ISC license is a permissive free software license published by the Internet Software Consortium, now called Internet Systems Consortium. It is functionally equivalent to the simplified BSD and MIT licenses, but without language deemed unnecessary following the Berne Convention. [Wikipedia](https://en.wikipedia.org/wiki/ISC_license)

> CC0-1.0
> You can copy, modify, distribute and perform the work, even for commercial purposes, all without asking permission.

Reply:
cargo-nextest uses crates with these licenses.

I want to add it to dev dependencies but there was one without a license which is icky, because its dev dependencies Ares does not rely on them however i cant tell cargo deny not to scan dev dependencies